### ISSUE TITLE
 Firefox 29 has no add-on bar

### ISSUE THREAD

**Issue by aniol** [surdu/deviantAnywhere-chrome#27]
In Firefox 29 with the new Australis interface the add-on bar has been removed. You can read more here: https://developer.mozilla.org/en-US/Firefox/Australis_add-on_compat
So deviantAnywhere is not shown if you don't install an addon to restore the addon bar, like The Addon Bar (restored): https://addons.mozilla.org/en/firefox/addon/the-addon-bar/

**Comment by kav2k** [comment]
It's a repository for the Chrome version of deviantAnywhere, though..

**Comment by aniol** [comment]
Look kav2k: https://github.com/postal2600/deviantAnywhere-chrome/issues?labels=Firefox

**Comment by ghost** [comment]
Yeah, for now the issue tracker tracks also the Firefox problems. I'm not happy to hear this, as I don't have the time to write the new FF version for this from scratch :(.
